docs/src/examples.md

```diff
@@ -71,14 +71,16 @@ To read the data directly use the function [`PSRI.file_to_array`](@ref) or [`PSR
 ```@example rw_file
 data_from_file = PSRI.file_to_array(
         PSRI.OpenBinary.Reader, 
-        FILE_PATH
+        FILE_PATH,
+        false
```

Comment:
Both examples should not change, you could add a new section on the md explaining the use case of `header = true`